modules/csvtk/concat/main.nf

```diff
@@ -0,0 +1,48 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+params.options = [:]
+options        = initOptions(params.options)
+
+process CSVTK_CONCAT {
+    tag "$meta.id"
+    label 'process_low'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::csvtk=0.23.0" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/csvtk:0.23.0--h9ee0642_0"
+    } else {
+        container "quay.io/biocontainers/csvtk:0.23.0--h9ee0642_0"
+    }
+
+    input:
+    tuple val(meta), path(csv)
+    val(in_format)
+    val(out_format)
+
+    output:
+    tuple val(meta), path("*.${out_format}"), emit: table
```

Comment:
```suggestion
    tuple val(meta), path("*.${out_format}"), emit: csv
```

Reply:
I need to fix this, because currently you can give `tsv`, `csv`, or any character to delimit things

If you give for example `=` to delimit things, your output file will be `*.=` which makes no sense.